src/contact.ts

```diff
@@ -222,12 +222,17 @@ export class Contact implements Sayable {
   }
 
   /**
-   * find contact by `name`(NickName) or `remark`(RemarkName)
+   * find contact by `name` or `alias`
    */
-  public static async findAll(queryArg?: ContactQueryFilter): Promise<Contact[]> {
+  public static async findAll(queryArg?: ContactQueryFilter | {remark: string | RegExp}): Promise<Contact[]> {
```

Comment:
Ok, let me tell u one more:

Next time, if u still only know what instead of knowing why, and make stupid argue like this, then kick your ass 10 times by yourself.